.idea/discord.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="DiscordProjectSettings">
+    <option name="show" value="ASK" />
+    <option name="description" value="" />
+  </component>
+</project>
```

Comment:
Raczej katalogu `.idea` nie wrzuca się do repo (to twoje ustawienia IDE) - warto o tym pamiętać i dodać go do `.gitignore` ;)